itest/rust/Cargo.toml

```diff
@@ -12,6 +12,7 @@ crate-type = ["cdylib"]
 [features]
 default = []
 codegen-full-experimental = ["godot/codegen-full", "godot/experimental-godot-api"]
+experimental-threads = ["godot/experimental-threads"]
```

Comment:
I guess this is for the `rid_test.rs` that accidentally depends on this?

Reply:
yeah